runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -359,26 +360,17 @@
     @Getter
     private volatile boolean isShutdown = false;
 
-    /**
-     * Metrics: meter (counter), histogram.
-     */
-    private static final String mp = "corfu.runtime.";
-    @Getter
-    private static final String mpASV = mp + "as-view.";
-    @Getter
-    private static final String mpLUC = mp + "log-unit-client.";
-    @Getter
-    private static final String mpCR = mp + "client-router.";
-    @Getter
-    private static final String mpObj = mp + "object.";
     @Getter
     private static MetricRegistry defaultMetrics = new MetricRegistry();
     @Getter
+    @Setter
     private MetricRegistry metrics = new MetricRegistry();
 
-    public CorfuRuntime setMetrics(@NonNull MetricRegistry metrics) {
-        this.metrics = metrics;
-        return this;
+    /** Initialize defaultMetrics */
```

Comment:
You need to describe what defaultMetrics are and why this is static

Reply:
This is a registry and adding comments to describe.